chartlets.js/packages/lib/src/actions/handleHostStoreChange.ts

```diff
@@ -20,6 +20,8 @@ export interface PropertyRef extends ContribRef, CallbackRef, InputRef {
   property: string;
 }
 
+const processedinputValuess = new Map<string, boolean>();
```

Comment:
No good design to store stuff in loose global variables. No lifecycle, no resource mgt.

If at all, it is `processedInputValuess ` not `processedinputValuess`.